firmware/vcv_plugin/export/src/module_widget_adaptor.hh

```diff
@@ -140,6 +140,20 @@ struct ModuleWidgetAdaptor {
 			pr_err("Error: can't add a null SvgWidget\n");
 	}
 
+	void addText(rack::ui::TextField *widget) {
+		if (widget) {
+			widget->element = make_element(widget);
+			assign_element_fields(widget, "");
+
+			ElementCount::Indices indices = clear();
+			indices.light_idx = 0; // widget->firstLightId;
```

Comment:
What should this be? I might want to add multiple LCD displays. I don't have a LIGHT in the VCV sense assigned for these displays, should i?